g-local_vce_detailed.md

```diff
@@ -318,3 +318,32 @@ The following flags may also be used with the `docker run` command:
 | Automatically remove container when it exits  | `--rm`                                                   |
 | Run container in background      | `-d/--detach`                                            |
 | Set environment variable inside  container     | `-e/--env ENV_VARIABLE="ENV VALUE"`                      |
+## Updating the Docker application
+It is good practice to keep your version of Docker Desktop up to date. If there are updates to Docker Desktop, the *Update to latest* button will appear to the left of the Docker Desktop search toolbar, {numref}`docker_desktop_update`.
+
+```{figure} md_assets/media/docker_update.png
+:name: docker_desktop_update
+:width: 5.44791666666667in
+:height: 2.041666666667in
+
+Checking for updates in Docker Desktop
+
+```
+It is possible to update to the latest version by doing the following.
+- Click on the *Update to latest* button. This will take you to the *Softwares updates* page.
+- Click on *Download update* and then *Update and restart*.
+
+Note: we strongly recommend that you **do not update the software within the VCE unless told to do so by the module team**. Software updates and upgrades occasionally introduce changes that result in undesirable software behaviour, sometimes known as ‘breaking changes’, that cannot always be predicted in advance.
+
+## Uninstalling the Docker application
+When you no longer require the Docker application to run the VCE (or VCEs for any other module you study), it is recommend that that you uninstall the application. 
+### Microsoft Windows
+For Windows devices
+- click on *Start* then *Settings* then *Apps*
+- In the list of Apps & Features, then scroll down to find *Docker Desktop*
+- Click on *Docker Desktop*  and select *Uninstall*.
+### Apple macOS
+For Mac devices
```

Comment:
devices:

There is also inconsistency in mac and win lists regarding capitalisation of first letter...

*Who needs editors when you have collaborative comment and review tools?! ;-)*

Reply:
Indeed these tools are helping to iron out the typos and inconsistencies. I'm certainly aware that I'm much better spotting minor issues in others stuff than my own.

Reply:
Re: spotting other's issues (others'?!;-)  - yes, me too...

Reply:
if you make the changes in the branch you have issued the PR from, the updates should appear here...

Reply:
As you can tell - I'm still learning to work in GitHub.  I have done those changes in my branch and re-done the pull request. Is that right?